SasquatchMac/SasquatchMacSwift/AppDelegate.swift

```diff
@@ -5,6 +5,14 @@ import AppCenterAnalytics
 import AppCenterCrashes
 import AppCenterPush
 
+enum StartupMode: Int {
+    case APPCENTER
```

Comment:
Can we use camel case? `appCenter`, `oneCollector`, `both`, `none`, and `skip`?

Reply:
Resolved in new commit.